lib/holidays.rb

```diff
@@ -160,6 +160,50 @@ def next_holidays(holidays_count, options, from_date = Date.today)
       UseCaseFactory.next_holiday.call(holidays_count, from_date, date_driver_hash, regions, observed, informal)
     end
 
+    # Get all holidays occuring from date to end of year, inclusively. 
+    #
+    # Returns an array of hashes or nil. 
+    #
+    # Incoming arguments are below: 
+    # [<tt>options</tt>]  One or more region symbols, <tt>:informal</tt> and/or <tt>:observed</tt>.
+    # [<tt>from_date</tt>]    Ruby Date object. This is an optional param, defaulted today.
+    #
+    # ==== Example
+    #   Date.today
+    #   => Tue, 23 Feb 2016
+    # 
+    #   regions = [:ca_on]
+    # 
+    #   Holidays.year_holidays(regions)
+    #   => [{:name=>"Good Friday",...},
+    #       {name=>"Easter Sunday",...},
+    #       {:name=>"Victoria Day",...},
+    #       {:name=>"Canada Day",...},
+    #       {:name=>"Civic Holiday",...},
+    #       {:name=>"Labour Day",...},
+    #       {:name=>"Thanksgiving",...},
+    #       {:name=>"Remembrance Day",...},
+    #       {:name=>"Christmas Day",...},
+    #       {:name=>"Boxing Day",...}]
+    def year_holidays(options, from_date = Date.today)
```

Comment:
Nope, I 100% agree that a little duplication is better than a bad combination. Leave it as-is. 👍